app/assets/javascripts/application.js

```diff
@@ -3,11 +3,11 @@
 
 //= require jquery
 //= require jquery_ujs
-//= require jquery-ui/datepicker
-//= require jquery-ui/draggable
-//= require jquery-ui/droppable
-//= require jquery-ui/tabs
-//= require jquery-ui/dialog
+//= require jquery-ui/widgets/datepicker
```

Comment:
I'm not sure it's a big deal, but the date picker widget has some styling changes. It's much flatter. I think I'm cool with it, but I wanted to mention it.

![screen shot 2017-03-02 at 7 31 35 pm](https://cloud.githubusercontent.com/assets/1099111/23534875/52bf6cf0-ff80-11e6-8bdb-a42a13313d81.png)
![screen shot 2017-03-02 at 7 30 54 pm](https://cloud.githubusercontent.com/assets/1099111/23534874/52bf2574-ff80-11e6-9a58-308e423f4acc.png)